dockerfiles/sfpowerscripts.Dockerfile

```diff
@@ -90,9 +99,16 @@ RUN echo 'y' | sfdx plugins:install sfdmu@4.13.0
 RUN echo 'y' | sfdx plugins:install sfpowerkit@4.2.5
 
 
+# install sfpowerscripts
 RUN echo 'y' | sfdx plugins:install @dxatscale/sfpowerscripts@$SFPOWERSCRIPTS_VERSION
 
 
+# Install PMD
```

Comment:
Reorder, as PMD version does not change most frequently

Reply:
not sure whether it matters, as we are not using cache anymore